Dockerfile.rhel7

```diff
@@ -1,7 +1,8 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
```

Comment:
i think this is technically .rhel8 as that's the base image is comes from, admiting it's preexisting

Reply:
:man_shrugging: I'm leaving this consistent with all the rest.